openmetadata-ui/src/main/resources/ui/src/components/SampleDataTable/SampleDataTable.component.tsx

```diff
@@ -153,27 +154,30 @@ const SampleDataTable: FunctionComponent<Props> = ({ sampleData }: Props) => {
             </tbody>
           </table>
         ) : (
-          <div className="tw-flex tw-justify-center tw-font-medium tw-items-center tw-border tw-border-main tw-rounded-md tw-p-8">
-            <Empty
-              description={
-                <>
-                  <p>No sample data available</p>
-                  <p className="tw-mt-2">
-                    To view Sample Data, run the Profiler Ingestion. Please
-                    refer to this doc to schedule the{' '}
-                    <Link
-                      className="tw-ml-1"
-                      target="_blank"
-                      to={{
-                        pathname:
-                          'https://docs.open-metadata.org/openmetadata/ingestion/workflows/profiler',
-                      }}>
-                      Profiler Ingestion
-                    </Link>
-                  </p>
-                </>
-              }
-            />
+          <div className="tw-flex tw-flex-col tw-justify-center tw-font-medium tw-items-center tw-p-8">
+            <div className="tw-mt-12">
+              <img alt="No Service" src={NoDataFoundPlaceHolder} width={120} />
+            </div>
+            <div className="tw-mt-8 tw-max-w-x tw-text-center">
+              <Typography.Paragraph style={{ marginBottom: '4px' }}>
+                {' '}
+                No sample data available
+              </Typography.Paragraph>
+              <Typography.Paragraph>
+                {' '}
+                To view Sample Data, run the Profiler Ingestion. Please refer to
+                this doc to schedule the{' '}
+                <Link
+                  className="tw-ml-1"
+                  target="_blank"
+                  to={{
+                    pathname:
+                      'https://docs.open-metadata.org/openmetadata/ingestion/workflows/profiler',
```

Comment:
Can we move these links to constant and use that constant here?

Reply:
Updated